### ISSUE TITLE
Option to invert highlight current cell

### ISSUE THREAD

**Issue by jnsebgosselin** [spyder-ide/spyder#4776]
I kind of find it distracting to work on a code that is highlighted. So I usually keep the "Highlight current cell" option turned off.

I think it would it be interesting to have an option in the editor preferences to invert the highlighting of the current cell. Here is how this could look like:

![invert_cell_highlight](https://user-images.githubusercontent.com/10170372/28399371-9ce098fa-6cda-11e7-84af-93343bd02b70.png)

It is almost possible to do the same thing in Syntax coloring by inverting current cell highlight and background color, but it doesn't look as good.

**Comment by ccordoba12** [comment]
@goanpeca, @rlaverde, what do you guys think?

**Comment by goanpeca** [comment]
I don't think I completely understand :-p

**Comment by ccordoba12** [comment]
@jnsebgosselin want cells to not be highlighted. Instead, the text around them would be "highlighted".

**Comment by goanpeca** [comment]
Hmmm interesting, I think it makes sense, or at least is more readable? the idea o the cell being to ficus on that text?

**Comment by jnsebgosselin** [comment]
@ccordoba12 yes the idea is to make the current cell not highlighted, while making the rest of the text highlighted. And yes @goanpeca, the goal is to make the code in the current cell more readable, while still providing a visual distinction between the current cell and the surrounding text. 

I've made a side by side comparison in the figure below. Personally, I find it easier on the eye and less distracting to work in the editor on right with "inverted highlighting" on.

![inver_highlight_side_comparison](https://user-images.githubusercontent.com/10170372/28506717-48ddc68c-6ffb-11e7-9ef7-c4ae3626130c.png)

**Comment by rlaverde** [comment]
I agree with this change, It'll be an option? or the default behavior?

**Comment by ccordoba12** [comment]
Ok, I'm -1 on it, but the majority wins!

It'll be a new option because the current behavior is the same as Matlab's one, and we want to be as close to Matlab as possible,

**Comment by goanpeca** [comment]
Just wondering if this is something you still find interesting to work on @jnsebgosselin 

Also, you seem to prefer the light theme over the dark theme I believe?

**Comment by levibaguley** [comment]
@jnsebgosselin has this been implemented?

**Comment by jnsebgosselin** [comment]
@goanpeca If you still think this is a good idea, I can work on this. Should not be that hard to do... I hope :P

@levibaguley no it is not implemented yet, sorry.

**Comment by goanpeca** [comment]
I am +1 on this. Would like to hear what other @spyder-ide/core-developers have in mind :-)

**Comment by jitseniesen** [comment]
I don't understand the picture. To me, the picture on the right looks like the current behaviour, but this is supposed to be the new behaviour? :confused:

**Comment by jnsebgosselin** [comment]
> 
> 
> I don't understand the picture. To me, the picture on the right looks like the current behaviour, but this is supposed to be the new behaviour? 😕

No, current behaviour is the picture on the left.

![cells_highlight](https://user-images.githubusercontent.com/10170372/81862383-a576fd00-9537-11ea-8423-e41b11e584aa.gif)

**Comment by ccordoba12** [comment]
I'm +1 on this.

**Comment by jitseniesen** [comment]
> No, current behaviour is the picture on the left.

Ah, something (probably me) changed the colours in my preferences. Got it now. It seems a good idea, so go ahead.
